yb-voyager/cmd/live_migration.go

```diff
@@ -65,8 +65,9 @@ func streamChanges() error {
 	if err != nil {
 		return fmt.Errorf("failed to initialize stats reporter: %w", err)
 	}
-	defer finalizeStats()
+
 	if !disablePb {
+		defer finalizeStats()
```

Comment:
Probably, doesn't matter right now, but it would be read better to call this _after_ calling `go statsReporter.ReportStats(ctx), i.e. you initialize, then defer finalize (as opposed to the other way around)

Reply:
For example, if we add any error handling(where we return err) in future between this statement, and `go statsReporter.ReportStats()` , we can run into the same issue

Reply:
Hmm got it makes sense